scripts/tools/combine-tools.ts

```diff
@@ -0,0 +1,205 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
Instead of disabling it for the whole file, can we somehow add it in respective lines or refactor the script in such a way as to avoid this rule override?

Reply:
Done!